multichain-testing/scripts/fast-usdc-tool.ts

```diff
@@ -205,7 +205,6 @@ const main = async () => {
         instancePath: [contractName],
         callPipe: [['makeDepositInvitation']],
       },
-      // @ts-expect-error 'NatAmount' vs 'AnyAmount'
```

Comment:
Why is this eslint change affecting TypeScript?

Reply:
It may be because of some changes to `tsconfig`, but I couldn't figure out how to keep this line without this error:

```
scripts/fast-usdc-tool.ts:208:7 - error TS2578: Unused '@ts-expect-error' directive.

208       // @ts-expect-error 'NatAmount' vs 'AnyAmount'
          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```